kratos/processes/calculate_discontinuous_distance_to_skin_process.h

```diff
@@ -313,6 +318,15 @@ class KRATOS_API(KRATOS_CORE) CalculateDiscontinuousDistanceToSkinProcess : publ
         array_1d<double,3>& rPlaneBasePointCoords,
         array_1d<double,3>& rPlaneNormal);
 
+        /**
+     * @brief Checks and replaces the values of the ELEMENTAL_DISTANCES vector that are
+     * zero. The values are replaced by an epsilon (whose sign depends on a flag)
+     * that is a fixed factor from the double precision. Can be deactivated by a flag.
+     * @param rElementalDistances array containing the ELEMENTAL_DISTANCES values
+     */
+    void ReplaceZeroDistances(
+        Vector& rElementalDistances);
```

Comment:
```suggestion
    void ReplaceZeroDistances(Vector& rElementalDistances);
```
(one line for single argument signature).